http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -378,9 +378,11 @@ class ClusterMonitorActor(val cluster: Cluster,
     val result = cluster.dataprocInfo.stagingBucket match {
       case Some(stagingBucketName) => {
         for {
+          _ <- IO(logger.info("persisting errors")).unsafeToFuture()
```

Comment:
I like to put `${cluster.projectNameString}` in every log message, if I can -- helps with traceability

Reply:
these log lines weren't there before...I'll remove them for now....regarding your comment, I think it's a great point, and I think we should create a ticket about adding logging in json format (with clusterName, and googleProject) which can be indexed in kibana